core/modules/views/handlers/views_handler_field_bulk_form.inc

```diff
@@ -217,32 +222,4 @@ class views_handler_field_bulk_form extends views_handler_field {
 
     return $options;
   }
-
-  /**
-   * Determine the base table of the bulk operation field.
-   */
-  protected function get_entity_type() {
```

Comment:
This removal is a low risk change, as probably nothing extends views_handler_field_bulk_form.

But for the sake of BC, can you restore it and let it just return `$this->entity_type;`?